src/abstractMatrix.js

```diff
@@ -980,6 +980,33 @@ export default function AbstractMatrix(superCtor) {
       return dot;
     }
 
+    /**
+         * Returns the element-wise product between this and other
+         * @param {Matrix} other
+         * @return {Matrix}
+         */
+    emul(other) {
+      other = this.constructor.checkMatrix(other);
+      if (this.rows !== other.rows || this.columns !== other.columns) {
+        // eslint-disable-next-line no-console
+        console.warn('Shape of left matrix are not equal to the shape of right matrix.');
```

Comment:
I would rather throw an error

Reply:
I just left warning messages like the other functions do.
I agree with your opinion. I'll change the code to throw an error :)